providers/azure/resources/azure.lr

```diff
@@ -1699,6 +1699,22 @@ private azure.subscription.cloudDefenderService @defaults("defenderForServers de
   monitoringAgentAutoProvision() bool
   // List of Defender for Server components and whether they are enabled
   defenderForServers() dict
+  // Whether Microsoft Defender for App Services is enabled
+  defenderForAppServices() dict
+  // Whether Microsoft Defender for SQL Servers on Machines is enabled
+  defenderForSqlServersOnMachines() dict
+  // Whether Microsoft Defender for Azure SQL Databases is enabled
```

Comment:
```suggestion
  // Whether Microsoft Defender for Azure SQL is enabled
```

Reply:
The focus is on " Azure SQL Databases" here